yb-voyager/cmd/analyzeSchema.go

```diff
@@ -50,6 +50,35 @@ type sqlInfo struct {
 	formattedStmt string
 }
 
+var (
+	anything                       = `.*`
+	ws                             = `[\s\n\t]*`
+	ident                          = `[a-zA-Z0-9_."]+`
+	ifExists                       = opt("IF EXISTS")
+	ifNotExists                    = opt("IF NOT EXISTS")
+	commaSeperatedStrings          = `[^,]+(?:,[^,]+){0,}`
+	commaSeperatedStringsMoreThan1 = `[^,]+(?:,[^,]+){1,}`
```

Comment:
it is `more than one comma separated`.

Reply:
That is not what the implementation does:
<img width="435" alt="image" src="https://user-images.githubusercontent.com/98899964/216583510-0adc3db2-6e36-4b7e-93b9-25daffcf2d75.png">

Reply:
Yes, I think I am explaining in wrong way. It is to capture more than 1 comma separated  string. If it will be 1 then it won't be comma separated. 
Basically what I am trying to say is that comma separated will be catched with second one and if it won't be comma separated which means only single string then can also be catched by first. 
Oh got it! My naming changed the meaning of them. Your's makes sense. Thanks! I will change it.